4-functions.js

```diff
@@ -0,0 +1,37 @@
+// Task 1 - Fuel calculator: for a given MPG and distance, calculate the number of gallons of fuel needed for a trip.
+var mpgVal = 55;
+var distanceVal = 110;
+
+var calculateNumberOfGallons = function (mpg, distance) {
+    return distance/ mpg;
+};
+
+console.log(calculateNumberOfGallons(mpgVal, distanceVal));
```

Comment:
Id just hard-code the test values, not define them as separate variables. Not really part of the program, just ad-hoc test code. It also confuses the scope for other devs as you've [reasonably] named the function pramaters the same thing.